pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -630,6 +630,9 @@ func (asw *actualStateOfWorld) GetVolumesToReportAttached() map[types.NodeName][
 }
 
 func (asw *actualStateOfWorld) GetNodesToUpdateStatusFor() map[types.NodeName]nodeToUpdateStatusFor {
+	asw.Lock()
```

Comment:
I only see this function being called in tests, in serial.

Reply:
Agree @msau42.